src/test/resources/StandaloneConfigTest/invalidFormats_config.json

```diff
@@ -0,0 +1,30 @@
+{
+	"ignoreGlobList": ["collated**"],
+	"formats": ["java", " && rm -rf /"],
```

Comment:
please don't use malicious code in test... although it doesn't work in this scenario, but what if it works?